src/tls/Makefile.OCaml

```diff
@@ -144,7 +142,12 @@ extract/OCaml/PKI.cmx: extract/mlstubs/PKI.ml extract/OCaml/TLSConstants.cmx
 extract/OCaml/BufferBytes.cmx: extract/mlstubs/BufferBytes.ml
 
 %.cmx:
-	$(OCAMLOPT) -w -8-11-26-20 -c $< -o $@
+ifdef VERBOSE
+	@echo "\033[0;32m=== Compiling $@ ...\033[;37m"
+endif
+	$(subst \\,, \
```

Comment:
can you put a comment here explaining the `$(subst`?

Reply:
Remove backslashes from the command-line echos, so they are easier to read/follow. Special request from @fournet :)